bullet_train/docs/upgrades/options.md

```diff
@@ -0,0 +1,93 @@
+# Options For Upgrading Your Bullet Train Application
+
+## Quick Links
+
+* [The YOLO Method (The original upgrade method)](/docs/upgrades/yolo.md)
+* [The Stepwise Method - For `1.4.0` and above](/docs/upgrades)
+* [Upgrade from any version to `1.4.0`](/docs/upgrades/yolo-140.md)
+* [Upgrade from any version to `1.3.0`](/docs/upgrades/yolo-130.md)
+* [Upgrade from `1.3.x` to `1.4.0`](/docs/upgrades/yolo-140.md)
+
+## About the upgrade process
+
+The vast majority of Bullet Train's functionality is distributed via Ruby gems, but those gems depend on certain
+hooks and initializers being present in the hosting application. Those hooks and initializers are provided by the
+[starter repository](https://github.com/bullet-train-co/bullet_train).
+
+Starting in mid-August of 2023 we began to iterate on how we publish gems and how we ensure that the starter repo
+will get the version of the gems that it expects.
+
+Starting with version `1.3.0` we began to explicitly update the starter repo every time that we released new
+versions of the Bullet Train gems. Unfortunately, at that time we were only making changes to `Gemfile.lock`
+which kind of hides the dependencies, and is often a source of merge conflicts that can be hard to sort out.
+
+Starting with version `1.4.0` we added explicit version numbers for all of the Bullet Train gems to `Gemfile`,
+which is less hidden and is not as prone to having merge conflicts as `Gemfile.lock`.
+
+As a result of these changes, there are a few different ways that you might choose to upgrade your application
+depending on which version you're currently on.
+
+## How to find your current version
```

Comment:
You could also just do

```sh
$ bundle show bullet_train
/Users/kolbrich/.asdf/installs/ruby/3.2.2/lib/ruby/gems/3.2.0/gems/bullet_train-1.3.17
```

or 

```sh
$ bundle show | grep bullet_train
  * bullet_train (1.3.17)
  * bullet_train-api (1.3.17)
  * bullet_train-fields (1.3.17)
  * bullet_train-has_uuid (1.3.17)
  * bullet_train-incoming_webhooks (1.3.17)
  * bullet_train-integrations (1.3.17)
  * bullet_train-integrations-stripe (1.3.17)
  * bullet_train-obfuscates_id (1.3.17)
  * bullet_train-outgoing_webhooks (1.3.17)
  * bullet_train-roles (1.3.17)
  * bullet_train-routes (1.0.0)
  * bullet_train-scope_questions (1.3.17)
  * bullet_train-scope_validator (1.3.17)
  * bullet_train-sortable (1.3.17)
  * bullet_train-super_load_and_authorize_resource (1.3.17)
  * bullet_train-super_scaffolding (1.3.17)
  * bullet_train-themes (1.3.17)
  * bullet_train-themes-light (1.3.17)
  * bullet_train-themes-tailwind_css (1.3.17)
```

Reply:
Great suggestion @olbrich, I'm going to simplify this section.